src/drivers/px4io/px4io.cpp

```diff
@@ -542,7 +525,7 @@ int PX4IO::init()
 	}
 
 	/* try to claim the generic PWM output device node as well - it's OK if we fail at this */
-	if (_param_sys_hitl.get() <= 0) {
+	if (_param_sys_hitl.get() <= 0 && _param_sys_use_io.get() == 1) {
```

Comment:
Given that legacy configurations might have set param values of 2 or 3, should we just check here for > 1

Reply:
The values 2 and 3 are not yet on master, there're added here: https://github.com/PX4/PX4-Autopilot/pull/16444/files#diff-6164e5c30f73f0b3e3c9265b27941dcfc72a9c7ead8ce2236f65c5912c34e015R52

Reply:
Got it.